test/runtime/fcntl.c

```diff
@@ -0,0 +1,89 @@
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+#include <stdio.h>
+#include <fcntl.h>
+#include <errno.h>
+
+/* expect = 0 for success, errno otherwise */
+void getlk(int fd, struct flock *lock, int expect)
```

Comment:
Minor nit, but these three functions are all nearly identical and could be folded into one with parameterized op (aside the check for F_UNLCK).

Reply:
Perfect! I just squashed all three functions into one specific to these three `*LK` commands -- later on, more tests can be added for the rest of the commands.